test/hummingbot/strategy/amm_arb/test_utils.py

```diff
@@ -35,10 +36,14 @@ def get_order_price(self, trading_pair: str, is_buy: bool, amount: Decimal) -> D
 
 
 class AmmArbUtilsUnitTest(unittest.TestCase):
+
     def test_create_arb_proposals(self):
+        asyncio.get_event_loop().run_until_complete(self._test_create_arb_proposals())
```

Comment:
I think it would be better to use asyncio.run()

Reply:
agree, since we are now on python 3,8, this is now possible.